### PR TITLE
Internal: Remove duplicate array key profile_class in EmailTokenServices

### PR DESCRIPTION
**Pull request by ribel**
**IMPORTANT:**
Should only be merged in `11.10.x` and `11.9.x`

## Problem
During cherry-pick of the PR https://github.com/goalgorilla/open_social/pull/3459 to `11.10.x` and `11.9.x` there was conflict. After resolving duplicate key `#profile_class` was added to the `getUserPreview` method of `EmailTokenServices`

## Solution
Remove duplicate `#profile_class` as it was done in the original PR #3459 

## Issue tracker
Internal.

## Theme issue tracker
N/A

## How to test
- Check if PHPStan pass

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Screenshots
image

## Release notes
Internal. See the problem part.

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by Kingdutch** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3468-xdlyj8mqm30ltioujkoshk3hydj0ltde.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/64ba5c3bd8b2f3192171cd2d

**Review by robertragas — APPROVED** [review]
Approved but waiting for the checks before I merge it

**Comment by robertragas** [comment]
Merged in 11.10.x and cherry-picked to 11.9.x

**Pull request merged**
